pantheon.yml

```
@@ -2,3 +2,4 @@ api_version: 1
 web_docroot: true
 build_step: false
 php_version: 8.1
+drush_version: 11
```

Oh btw, this gets rejected by Pantheon, just remembered as I was writing my blog post :laughing: 

Max it can go is 10. But if you have 11 installed in vendor folder, they'll use that instead.
```suggestion
drush_version: 10
```